lib/openhab/core/items/metadata/hash.rb

```diff
@@ -381,7 +226,6 @@ def inspect
             [value, to_h].inspect
           end
           remove_method :to_s
-          alias_method :to_s, :inspect
```

Comment:
why was this removed?

Reply:
I put this in EmulateHash. Also removed `remove_method :to_s` and the `:to_s` delegation at the top of this class. Hopefully that means it'll use the one in EmulateHash.